src/Coordination/KeeperStorage.cpp

```diff
@@ -917,6 +917,17 @@ struct KeeperStorageCreateRequestProcessor final : public KeeperStorageRequestPr
 
         std::vector<KeeperStorage::Delta> new_deltas;
 
+        if (zk_request->getOpNum() == Coordination::OpNum::CreateIfNotExists) {
+            auto & container = storage.container;
+            auto node_it = container.find(request.path);
+            if (node_it != container.end())
+            {
+                return new_deltas;
+            }
+            // if (storage.uncommitted_state.getNode(request.path) == nullptr)
```

Comment:
```suggestion
            // if (storage.uncommitted_state.getNode(request.path) != nullptr)
```
because you want to return if it DOES exist